packages/terra-framework-docs/src/terra-dev-site/doc/worklist-data-grid/example/DefaultWorklistDataGrid.jsx

```diff
@@ -1,22 +1,98 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
 import WorklistDataGrid from 'terra-worklist-data-grid';
 import gridDataJSON from './gridData.json';
 
 const DefaultWorklistDataGrid = () => {
+  const rowSelectionModeRef = useRef();
   const rowHeaderIndex = 0;
   const { cols, rows } = gridDataJSON;
+  const [selectedRows, setSelectedRows] = useState([]);
+  const [hasSelectableRows, setHasSelectableRows] = useState(false);
+
+  const determineSelectedRows = (allRowsSelected, userSelectedRow) => {
+    if (!userSelectedRow) {
+      return [];
+    }
+
+    let remainingSelectedRow = [];
+    if (allRowsSelected) {
+      remainingSelectedRow = userSelectedRow;
+    } else if (selectedRows.includes(userSelectedRow[0])) {
+      // Row Deselect so remove this rowId.
+      remainingSelectedRow = selectedRows.filter(e => (e !== userSelectedRow[0]));
+    } else {
+      // Row Selected so add this rowId.
+      remainingSelectedRow = remainingSelectedRow.concat(selectedRows);
+      remainingSelectedRow.push(userSelectedRow[0]);
+    }
+    return remainingSelectedRow;
+  };
+
+  const clearRowSelection = () => {
+    // eslint-disable-next-line no-param-reassign
+    rows.forEach(r => { if (r.isSelected) { r.isSelected = false; } });
+    setSelectedRows([]);
+  };
+
+  const setRowSelectionMode = () => {
+    rowSelectionModeRef.current.checked = false;
+    setHasSelectableRows(false);
+    clearRowSelection();
```

Comment:
If we are removing the row selection column, we should not need to call clear row selection.